XenModel/XenAPI-Extensions/VM.cs

```diff
@@ -841,21 +841,22 @@ public enum VmTemplateType
             CoreOS = 6,
             Debian = 7,
             Gooroom = 8,
-            Linx = 9,
-            NeoKylin = 10,
-            Oracle = 11,
-            RedHat = 12,
-            SciLinux = 13,
-            Suse = 14,
-            Turbo = 15,
-            Ubuntu = 16,
-            YinheKylin = 17,
-            Citrix = 18,
-            Solaris = 19,
-            Misc = 20,
-            Snapshot = 21,
-            SnapshotFromVmpp = 22,
-            Count = 23  //bump this if values are added
+            Rocky = 9,

```

Comment:
Given previous commits in this area I realise that these don't need to preserve ID numbers of existing items.
How do we choose the order of these items? Some of it seems roughtly alphabetical.

Reply:
According to the comment, there is a certain order required only for the various template categories. I think it makes sense though to have alphabetical order within each category (when no other rules apply, e.g. in the case of Windows it makes sense to put legacy systems second). So, yes, it's probably best to move Rocky after RedHat.

Reply:
Move rocky after redhat